openshift_metrics/merge.py

```diff
@@ -212,8 +210,14 @@ def main():
     else:
         pod_report_file = f"Pod NERC OpenShift {report_month}.csv"
 
-    report_start_date = datetime.strptime(report_start_date, "%Y-%m-%d")
-    report_end_date = datetime.strptime(report_end_date, "%Y-%m-%d")
+    report_start_date = datetime.strptime(report_start_date, "%Y-%m-%d").replace(
+        tzinfo=UTC
+    )
+    report_end_date = datetime.strptime(report_end_date, "%Y-%m-%d").replace(tzinfo=UTC)
+
+    logger.info(
+        f"Generating report from {report_start_date} to {report_end_date + timedelta(days=1)} for {cluster_name}"
+    )
 
     if report_start_date.month != report_end_date.month:
         logger.warning("The report spans multiple months")
```

Comment:
Should this if clause be removed, since if the extra string is appended, calling `nerc_rates` with it will raise an date parsing error